proto/devices.proto

```diff
@@ -159,3 +161,21 @@ message PullDevicesMetadataResponse {
     google.protobuf.Timestamp change_time = 3;
   }
 }
+
+message GetDownloadDevicesUrlRequest {
+  // A query for filtering the devices that will be returned
+  Device.Query query = 1;
+  // Specifies the media type (or MIME type) of the file to be downloaded.
+  // Defaults to text/csv.
+  // Supported values are text/csv (currently).
```

Comment:
Redundant "(currently)", we don't need to state that input parameters may support extra values in the future.